docs/notebooks/qubit_mapping_guide.ipynb

```diff
@@ -0,0 +1,796 @@
09t/5Eyra3LoQ23A8WbT0r+uCxZs6ntj9X50ORtI9v+OC1xXM/kzKrkly27UFhhF7wuOaJlV7wHYB/0f//v/82tt96al156qdSl0IoMhWlFO8Klly9uBnCg6tmt4cNeS6zb3PBBfN3mPe9TluQfT2sYtbAvKStL3vGGhg/9LVHkOThtQDLYOjP7nM6HNoR+LVXkNfDWEftmsPLmkxrC35Yocv4nHJ2cfkLLjgPAvkuwdODZx/5U37898sgjpS4BoF2dfkKy9Pnkt3/es+1vebDlx3jLiOT43i3v1x6O6JJcMTq57RcNU732REufg9cclVw04tW3ozQGH5P8w/Dkx3P3vE9LXwOjBzWsWbUvOrQ8+ae/S77y/5JNL+5Zn5aef89uyYQzkg77wELmtI6jjz46H/vYxxrXXQFg/2fkEgCFdShrGLkxon/b7H/8yQ1rG+3LTuzT8OG6LRYZfk3P5F/O2r+uonYwOndow60tjBrYsFbVvnCFuF05pkfyr2e3zZTHnt2SK9+UHNG59fdN6Rx11FG54oorctRRR5W6FABaiXAJgL3SsUNy2ekNayK11siCikOSS0ft2WXs9wVDj02uPDvp0YrTlkb0b/hQ3WXni4+yjykrS8ad1LAO2aGttC5Wxw4NU87e8Yb9Y8TOcT2Tq85tCJpay6DKhn327NZ6+2TfsHbt2jz44INZu3ZtqUsBoJUIlwDYax3KGqYGfeTvkz5H7N2+qvokH3tzw6Xe9ycDjk4+Nn7vr2Z1WEXy3jOTy88wYml/c8YJycQ3JwP2chrnsUcm15yXnDN03x6x9EqVRyRX//3eB82dyhuCun8724ilA9WyZcvykY98JMuWLSt1KQC0EmsuAdBq+vVMrvmHZG5NwyXa//zcnvUrK0uGHdvw4XxQ5f71gfrlKg5J3nFqw1pUDy9KfvdM8uL2Pevb+/CGtXVe/9qWLxLOvuOow5IPjE0W/qXhd2DBX5I9u1Zsw6LVZwxq+F1oi2mW7aG8Y0PQ/PrXNvwOzH56z9di6t6lIZw9bWByuFAJAPYrwiUAWlV5x+SU/g23FWuSRSuTmtXJsueT2rUNH7TL0hBEHXtk0u/IhtFKPbqWtOxWdeyRDWtRnf+6ZMHyhvOvWZ2sWp+8tL1hVEfFocmxPRq2Pb5X8tpe+2+oRlMdypLqvg235zckf1rxt9fAuk3JtrqG8Oiwir/9DgyqTI7ey1F/+5KjDkveckpD0LRwRcPC/zWrk5Vrk7Wb/vo+UJZUH9PwXvCang3Pwf4aqgHAwU64BECb6dO94bbDp+5J1m5uGJVw9Xmlqqr9dDk0GfHahhsHp57dGkayHawOLU+G92u47dD4PlCR/PNZJSsNAGhFvh8CAADaTUVFRQYPHpyKiopSlwJAKzFyCQAAaDcDBgzIPffcU+oyAGhFRi4BAAAAUJhwCQAAaDfz58/P8OHDM3/+/FKXAkArES4BAADtpr6+Pi+99FLq6+tLXQoArcSaSwAAAEATXbp0yYYNG1ptf//f176TdRs35fCuXfLRf7lkp/utoUuXLq2yH1pOuAQAAAA0UVZWlq5du7ba/g7tVJFDX9qeQztVpGvXrjvdZ/9mWhwAAAAAhRm5BAAAtJsBAwbk/vvvT79+/UpdCgCtRLgEAAC0m4qKipxwwgmlLgOAVmRaHAAA0G6WL1+eT3ziE1m+fHmpSwGglQiXAACAdrNmzZp8//vfz5o1a0pdCgCtRLgEAAAAQGHCJQAAAAAKEy4BAAAAUJirxQEAAIVVVVW1aPsjjjgin/rUp3L66aenT58+bVQVAO1JuAQAALSbPn365IYbbih1GQC0ItPiAAAAAChMuAQAAABAYcIlAAAAAAoTLgEAAABQmHAJAAAAgMKESwAAAAAUJlwCAAAAoDDhEgAAAACFCZcAAAAAKEy4BAAAAEBhwiUAAAAAChMuAQAAAFCYcAkAAACAwoRLAAAAABQmXAIAAACgMOESAAAAAIUJlwAAAAAoTLgEAAAAQGHCJQAAAAAKEy4BAAAAUJhwCQAAAIDChEsAAAAAFCZcAgAAAKAw4dJeWLVqVa699toMGDAgFRUV6devX6655pps3rw5EyZMSFlZWaZNm1bqMgEAAADaTHmpC9hfzZ07N+eee25WrlyZrl27prq6OsuXL88tt9ySxYsXZ+nSpUmSk046qcSVAgAAALQdI5cKWLVqVcaPH5+VK1dm4sSJqa2tzWOPPZaVK1dmypQpueeee/L444+nQ4cOGTp0aKnLBQAAAGgzwqUCrrrqqtTU1OTqq6/OTTfdlG7dujW2XX/99amqqsq2bdsyaNCgdO7cuYSVAgAAALQt4VILzZ8/PzNmzEjv3r0zZcqUndrLysoyYsSIJE2nxC1btixXXXVVTj/99HTp0iVlZWVZvHjxTv33dDsAAACAfYFwqYWmT5+eurq6TJgwIV26dGl2m06dOiVpGi499dRTmTFjRrp37543vvGNu9z/nm4HAAAAsC+woHcLPfTQQ0mSMWPG7HKbZcuWJWkaLp155plZuXJlkuSOO+7Igw8+2GzfPd2uJUaOHJna2tq93g/A3hr38UfT5Yg+WVG7Isce+/pSlwOUgPcB4GB10T99OF27Hf7X979jd7p/oDvYz39/UFlZmTlz5hTqK1xqoSVLliRJ+vfv32z71q1bM3v27CRNw6UOHfZskNiebtcStbW1Wb58eavvF6Cltm/f3viv9yU4OHkfAA5WdX99/6v76/vfK+8f6A728z/QCZdaaOPGjUmSLVu2NNt+5513Zu3atenZs2f69u3bnqXtUmVlZalLAEiSdOzYsfHffeU9Emhf3geAg1WHv77/dfjr+98r7x/oDvbz3x/sTXYgXGqhysrKrF+/PnPmzMnIkSObtNXU1GTSpElJkuHDh5eivGYVHdYG0No+dU+ydnPSp7JP4xRi4ODifQA4WE396vSs27Cx8f3vlfcPdAf7+R/oLOjdQmPHjk2STJ06NU8++WTj4
```

Comment:
How about "Example with larger circuits" instead of "Bigger circuits"? I think each section title is little too short, e.g. "Start", "Build" and so on.